docker/simplified_crontab

```diff
@@ -124,7 +124,7 @@ CRON_TZ=$TZ
 
 # update_lane_size - Update the cached sizes of all lanes.
 #   Frequency: Minute 2 of hour 10 (once daily)
```

Comment:
Could we change this to "#   Frequency: Minute 2 of hour 7 (once daily)"?

Reply:
Good catch!